.github/pull_request_template.md

```diff
@@ -12,33 +12,45 @@ Based on https://raw.githubusercontent.com/PyCQA/pylint/master/.github/PULL_REQU
 
 # ARTIQ Pull Request
 
-## Steps (Choose relevant, delete before submitting)
+## Steps (Choose relevant, delete irrelevant before submitting)
 
 ### All Pull Requests
 
-- [ ] Use correct spelling and grammar.
-- [ ] Update RELEASE_NOTES.md if there are noteworthy changes, especially if there are changes to existing APIs.
-- [ ] Close and/or update issues.
-- [ ] Check the copyright situation of your changes and sign off your patches (`git commit --signoff`, see [copyright](https://github.com/m-labs/artiq/blob/master/CONTRIBUTING.rst#copyright-and-sign-off)).
+- [x] Use correct spelling and grammar.
```

Comment:
Why is this one checked?

Reply:
As an example so people know how the checkboxes work (it's the first one). It's not immediately obvious just by looking at the boxes. Could add blank line:
` - [x] Sample checkbox`, but that's unnecessarily wordy.